accessibility-checker-engine/test/v2/checker/accessibility/rules/landmark_name_unique_ruleunit/example_4_mixed.html

```diff
@@ -0,0 +1,271 @@
+<!DOCTYPE html PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN" "http://www.w3.org/TR/html4/loose.dtd">
+<!--
+   /******************************************************************************
+     Copyright:: 2020- IBM, Inc
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+  *****************************************************************************/
+-->
+
+<html lang="en-US">
+
+<head>
+</head>
+
+<body id="body">
+  <main id="mainNode1">
+    <!-- This nav trigger the rule but does not pass or fail. It is not applicable. -->
+    <div id="TopLabel0" role="navigation" aria-label="TopLabel">
```

Comment:
```suggestion
    <div id="TopLabel0" role="navigation" aria-label="Top Label">
```

(should probably make the aria-label string understandable, even if it's just a test case)

Reply:
Fixed